providers/os/connection/local/statutil/stat.go

```diff
@@ -246,6 +248,84 @@ func (s *statHelper) unix(name string) (os.FileInfo, error) {
 	}, nil
 }
 
+func (s *statHelper) aix(name string) (os.FileInfo, error) {
+	path := ShellEscape(name)
+	var sb strings.Builder
+
+	// AIX does not ship with stat, therefore we use perl stat function to retrieve the same information as on linux
+	// Codes are taken from https://perldoc.perl.org/functions/stat
+	//0 dev      device number of filesystem
+	//1 ino      inode number
+	//2 mode     file mode  (type and permissions)
+	//3 nlink    number of (hard) links to the file
+	//4 uid      numeric user ID of file's owner
+	//5 gid      numeric group ID of file's owner
+	//6 rdev     the device identifier (special files only)
+	//7 size     total size of file, in bytes
+	//8 atime    last access time since the epoch
+	//9 mtime    last modify time since the epoch
+	//10 ctime    inode change time (NOT creation time!) since the epoch
+	//11 blksize  preferred block size for file system I/O
+	//12 blocks   actual number of blocks allocated
+	script := `perl -e '@a = stat(shift) or exit 2; $u = getpwuid($a[4]); $g = getgrgid($a[5]); printf("0%o:%s:%d:%s:%d:%d:%d", $a[2], $u, $a[4], $g, $a[5], $a[7], $a[9])'`
```

Comment:
The PR comment mentions Python. Is perl intended here?

Reply:
perl is the way to go. I update the comment.